lms/templates/ace_common/edx_ace/common/base_body.html

```diff
@@ -61,12 +61,12 @@
                 <table role="presentation" width="100%" align="left" border="0" cellpadding="0" cellspacing="0">
                     <tr>
                         <td width="70">
-                            <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://courses.xpro.mit.edu/static/mitxpro-theme/images/logo.png" width="130"
+                            <a href="{% with_link_tracking site_configuration_values.HOMEPAGE_URL %}"><img
+                                    src="https://rc.mitxonline.mit.edu/static/images/mitx-online-logo.png" width="130"
```

Comment:
I'm sure that we really shouldn't be using the `mitx-online-logo.png` from RC. It might be less possible that the logo is tweaked a lot but we cannot consider the logo image of RC to always be the source of truth for us.
So, I recommend the following options and you can opt for any of those based on what you think:
1. (_Simplest_) Replace the `src` to use logo URL of production instance e.g. https://mitxonline.mit.edu/static/images/mitx-online-logo.png - It will always pick the image from MITxOnline. The only drawback is that in the future if we change the domain we'll have to add updates to the theme since the URL will be hardcoded.
2. Make this image configurable as well like other Site config values, e.g. In this case we can place RC logo URL for RC instance of LMS and vice versa for prod.
3. If possible? Can we use the logo image from LMS's assets(https://github.com/mitodl/mitxonline-theme/blob/main/lms/static/images/logo.png) instead of using a URL from the marketing website?

Reply:
In addition to the above, The logo image currently looks contracted and we might need to update the style a bit so that the logo renders correctly. You might need to increase the width from `130px` but definitely, you can match the styling of the logo in the header. Attaching a screenshot of how it looks right now.
<img width="268" alt="Screenshot 2021-09-29 at 1 29 12 PM" src="https://user-images.githubusercontent.com/34372316/135232113-e5e897c6-ba84-442f-a283-3f5705ccd96b.png">